pandas/tests/io/test_excel.py

```diff
@@ -80,7 +83,6 @@ def df_ref(self):
                           parse_dates=True, engine='python')
         return df_ref
 
-    @td.skip_if_no("xlrd", "1.0.1")  # see gh-22682
```

Comment:
These didn't really solve the issue they were supposed to so removed to make things cleaner. Again I think the real issue occurs when defusedxml is installed in the right environment, but will have to delve further into which environments that is exactly (orthogonal to this PR)